tests/write_float/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+add_executable(
+    main
+    main.cpp 
+)
+target_include_directories(
+    main 
+    PUBLIC
+    ${CMAKE_SOURCE_DIR}/include
```

Comment:
why is this necessary? Use target_include_directories(driverapi PUBLIC include/) and this isn't necessary.

Reply:
https://github.com/mikex86/LibreCuda/pull/11/files#diff-683f56bff501ed400e016f5aafd273954e95a26a64fdf77b692709ffb494d10aR1-R8